tools/extract-lecture-structure/invoke_filewise.sh

```diff
@@ -0,0 +1,2 @@
+mkdir .tmp_structure
+make web_sources | xargs -n 1 bash -c 'pandoc "$0" --lua-filter=tools/extract-lecture-structure/filewise_pass.lua -o .tmp_structure/temp.md; cat .tmp_structure/temp.md >> .tmp_structure/temp_summary.md'
```

Comment:
Das sollte als Target ins Makefile?

Reply:
Die gesamte Generierung der structure.pdf ist jetzt als target in dem Makefile drin.